build/gulpfile.vscode.js

```diff
@@ -500,9 +499,13 @@ function packageTask(platform, arch, opts) {
 const buildRoot = path.dirname(root);
 
 // {{SQL CARBON EDIT}}
-gulp.task('vscode-win32-x64-azurecore', ['optimize-vscode'], packageAzureCoreTask('win32', 'x64'));
-gulp.task('vscode-darwin-azurecore', ['optimize-vscode'], packageAzureCoreTask('darwin'));
-gulp.task('vscode-linux-x64-azurecore', ['optimize-vscode'], packageAzureCoreTask('linux', 'x64'));
+gulp.task('vscode-win32-x64-azurecore', ['optimize-vscode'], packageExtensionTask('azurecore', 'win32', 'x64'));
+gulp.task('vscode-darwin-azurecore', ['optimize-vscode'], packageExtensionTask('azurecore', 'darwin'));
+gulp.task('vscode-linux-x64-azurecore', ['optimize-vscode'], packageExtensionTask('azurecore', 'linux', 'x64'));
+
+gulp.task('vscode-win32-x64-mssql', ['optimize-vscode'], packageExtensionTask('mssql', 'win32', 'x64'));
+gulp.task('vscode-darwin-mssql', ['optimize-vscode'], packageExtensionTask('mssql', 'darwin'));
+gulp.task('vscode-linux-x64-mssql', ['vscode-linux-x64-azurecore', 'optimize-vscode'], packageExtensionTask('mssql', 'linux', 'x64'));
```

Comment:
should we setup the build tasks to work the same on all platforms?  i.e. add 'vscode-linux-x64-azurecore', on macOS and Windows as well?

Reply:
I don't think so. vscode-linux-x64-mssql doesn't really depend on azurecore, they just need to run sequentially to avoid having lots of files open. This way things can go quicker on Mac/Windows